Bio/Application/__init__.py

```diff
@@ -212,7 +212,9 @@ def __init__(self, cmd, **kwargs):
         aliases = set()
         for p in parameters:
             if not p.names:
-                assert isinstance(p, _StaticArgument), p
+                if not isinstance(p, _StaticArgument):
+                    raise TypeError("Expected %s to be of type _StaticArgument"
+                                    % repr(p))
```

Comment:
This is neater using ``%r`` rather than ``%s`` and an explicit call of ``repr(...)``

```python
raise TypeError("Expected %r to be of type _StaticArgument" % p)
```

Reply:
That's a good point - completely forgot about `%r`. I noticed there are other explicit uses of `repr` that I think could be changed to use `%r` in [`Bio/Application/__init__.py`](https://github.com/biopython/biopython/blob/ad216e5302463c95971bb35a8cf49d6904cd064a/Bio/Application/__init__.py#L226). I could change those as well as part of this PR?

Reply:
Since you are working on error messages in this file anyway, the same PR should be fine. Equally a fresh PR for that issue would also be fine if you'd prefer.